cmd/support-notifications/res/docker/configuration.toml

```diff
@@ -32,9 +32,9 @@ File = '/edgex/logs/edgex-support-notifications.log'
   [Databases.Primary]
   Host = 'edgex-redis'
   Name = 'notifications'
-  Password = ''
+  Password = 'password'
   Port = 6379
-  Username = ''
+  Username = 'notifications'
   Timeout = 5000
   Type = 'redisdb'
```

Comment:
Does it matter that the service now uses Redis by default?

Reply:
It may, but having username and password set is consistent with all of the other TOML files
- https://github.com/edgexfoundry/edgex-go/blob/master/cmd/core-command/res/docker/configuration.toml
- https://github.com/edgexfoundry/edgex-go/blob/master/cmd/core-data/res/docker/configuration.toml
- https://github.com/edgexfoundry/edgex-go/blob/master/cmd/core-metadata/res/docker/configuration.toml
- et cetera

Setting this to blank caused a regression.